sys/kern/src/arch.rs

```diff
@@ -21,16 +21,16 @@ cfg_if::cfg_if! {
 
     if #[cfg(target_arch = "arm")] {
         #[macro_use]
-        pub mod arm_m;
-        pub use arm_m::*;
+        pub mod arm;
+        pub use arm::*;
     } else if #[cfg(target_arch = "riscv32")] {
         #[macro_use]
```

Comment:
At some point, I think we can put `rv32` and `rv64` under a single `rv` and create a hierarchy under that. But, I didn't do that now to make it easier for rv32 changes to be pushed upstream for review.